report/templates/benchmark.json

```diff
@@ -0,0 +1,19 @@
+{
+    "sample": [
+{% for sample in samples %}
+    {
+        "benchmark": "{{ benchmark.signature }}",
+        "sample": "{{ sample.id }}",
+        "status": "{{ sample.status }}",
+        "Compiles": "{{ sample.compiles }}",
+        "crashes": "{{ sample.crashes }}",
+        "coverage": "{{ sample.coverage | percent }}",
+        "coverage_diff": "{{ sample.result.line_coverage_diff }}",
+        "coverage_report": "{{ sample.result.coverage_report_path }}",
+        "stacktrace": "{{ sample.result.stacktrace_link }}",
```

Comment:
maybe just `output` to generalize this more. This would include both the libFuzzer output + stacktrace (if any).

Reply:
I see. BTW, I am adding a bit more code to https://github.com/google/oss-fuzz/pull/11700 to upload the stacktrace too.
Sorry about the frequent review requests an changes.

Reply:
Do we  need this?  I assume the  stacktrace/output is  already there somewhere  when we run  `web.py`  ?

Reply:
Would it be better to have a separate item to link to the `libFuzzer` output?
e.g., 
```
"output": "{{ sample.result.output_link }}",
```
This info is already available and need not to be uploaded to the new reproducer bucket directory.